db/seeds.rb

```diff
@@ -0,0 +1,15 @@
+# This file should contain all the record creation needed to seed the database with its default values.
+# The data can then be loaded with the bin/rails db:seed command (or created alongside the database with db:setup).
+#
+# Examples:
+#
+#   movies = Movie.create([{ name: "Star Wars" }, { name: "Lord of the Rings" }])
+#   Character.create(name: "Luke", movie: movies.first)
+
+Greeting.create(greeting:'Hello')
+Greeting.create(greeting:'Hola')
+Greeting.create(greeting:'konnichiwa')
+Greeting.create(greeting:'Ciao')
+Greeting.create(greeting:'Ni hao')
+Greeting.create(greeting:'Kaixo')
+Greeting.create(greeting:'Zadravo')
```

Comment:
- [ ] When I run `rails db:seed`, I get the following error. 

![image](https://user-images.githubusercontent.com/77106962/180625387-03fb48f4-f88f-4edb-9d7e-562118138962.png)

#### Kindly work on it so that a user can insert data into the database. 👍